proposals/date_between.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "date_between",
+    "summary": "Between comparison for dates and times",
+    "description": "By default, this process checks whether `x` is later than or equal to `min` and before or equal to `max`.\n\nIf `exclude_max` is set to `true` the upper bound is excluded so that the process checks whether `x` is later than or equal to `min` and before `max`.\n\nLower and upper bounds are not allowed to be swapped. So `min` MUST be before or equal to `max` or otherwise the process always returns `false`.",
```

Comment:
what about time-only intervals crossing midnight?

`between("01:00:00", min="22:00:00", max="04:00:00)`: invalid, false or true?

Reply:
I'd say invalid due to 

> Lower and upper bounds are not allowed to be swapped. So `min` MUST be before or equal to `max` or otherwise the process always returns `false`.